src/main/java/org/jabref/gui/entryeditor/OffersPreview.java

```diff
@@ -0,0 +1,14 @@
+package org.jabref.gui.entryeditor;
+
+public interface OffersPreview {
```

Comment:
Like the interface (composition over inheritance). Yet the name is precise, but boring. :wink: